src/task/event_queue.rs

```diff
@@ -0,0 +1,161 @@
+use neon_runtime::raw::Env;
```

Comment:
Also suggested on the RFC PR: I'd advocate `neon::event` instead of `neon::task` for this.